courseraresearchexports/commands/jobs.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from courseraresearchexports import exports
+from datetime import datetime
+from requests.exceptions import HTTPError
+from tabulate import tabulate
+import json
+import argparse
+import logging
+import sys
+
+
+def download(args):
+    """
+    Download a data export job using a job id.
+    """
+    job = exports.api.get(args.id)
+    exports.utils.download_export(job, args.dest)
+
+
+def create(args):
+    """
+    Create and send an data export request with Coursera.
+    """
+    export_job_json = exports.utils.build_export_job_json(**vars(args))
+    export_job_id = exports.api.create(export_job_json)
+
+    logging.info('Successfully created job %(id)s:\n%(json)s', {
+        'id': export_job_id,
+        'json': json.dumps(export_job_json, indent=2)
+    })
+
+
+def get(args):
+    """
+    Get the details and status of a data export request using a job id.
+    """
+    try:
+        export_job = exports.api.get(args.id)
+    except HTTPError as err:
+        logging.error('Export job %s could not found.\n%s', args.id, err)
+        sys.exit(1)
+
+    export_job_info = [['Export Job Id:', args.id]]
+
+    scope_id = exports.utils.get_scope_id_from_job(export_job)
+    export_job_info.append(['Scope Id:', scope_id])
+
+    export_type = export_job['exportType']
+    export_job_info.append(['Export Type:', export_type])
+
+    creation_time = datetime.fromtimestamp(
+        export_job['metadata']['createdAt'] / 1000.0).strftime('%c')
```

Comment:
seems like we have different date formats in different places, can we centralize this date format business? also, unix time is pretty hard to use, so lets avoid displaying it whenever possible.